README.md

```diff
@@ -54,6 +54,38 @@ make
 ./evcc
 ```
 
+### Debugging in VS Code
+
+#### evcc Core
+To debug a local evcc build in VS Code, add the following entry to your `launch.json`.
+You can adjust the referred configuration as needed to e.g. use your live configuration. 
+
+```json
+        {
+            "name": "Launch evcc local build with demo config",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}",
+            "args": ["-c", "${workspaceFolder}/cmd/demo.yaml"],
+            "cwd": "${workspaceFolder}",
+        },
+```
+
+#### Decorator
+Here's another `launch.json` configuration that can be used for specifically debugging the decorator.
```

Comment:
Ich finde das ein super harten Einstieg. Dafür müsste man erstmal wissen, was ein Decorator ist und was evcc damit tut? Einen Absatz dafür sprendieren? Dann kanns m.w auch in die README ;)

Reply:
Ja, da bin ich bei dir. Ich nutzte selbst weder `launch.json` noch Decorators in meinem Workflow. Eine kurze Erklärung, warum man das machen würde, wäre sicher hilfreich.

Reply:
Sinn und Zweck der Dinge zu erklären wäre noch gut. Auch die grundsätzliche Repo-Struktur. Beides nice to have. Den Decorator würde ich hier raus nehmen solange es nicht irgendeine Erklärung gibt, was das ist und wofür?